keyboards/hhkb_lite_2/keymaps/via/rules.mk

```diff
@@ -0,0 +1,13 @@
+VIA_ENABLE = yes
+
+BOOTMAGIC_ENABLE = lite
+
+# save space
+LTO_ENABLE = yes
+LEADER_ENABLE = no
+FAUXCLICKY_ENABLE = no
+MIDI_ENABLE = no
+BLUETOOTH_ENABLE = no
+KEY_LOCK_ENABLE = no
+TERMINAL_ENABLE = no
+MOUSEKEY_ENABLE = no
```

Comment:
```suggestion
VIA_ENABLE = yes
LTO_ENABLE = yes
```

Reply:
I'm not clear why you're suggesting this change. It is not functionally equivalent and will not result in working firmware because of the total size.

Reply:
`LEADER_ENABLE`, `FAUXCLICKY_ENABLE`, `MIDI_ENABLE`, `BLUETOOTH_ENABLE`, `KEY_LOCK_ENABLE` and `TERMINAL_ENABLE` are all either not specified in the keyboard-level rules.mk, or set to `no`, so there is no change for them.
`BOOTMAGIC_ENABLE = lite` is implied by `VIA_ENABLE = yes`, so no change there either.
Mousekeys are now supported in the latest version of VIA so they should be enabled if possible - with LTO enabled I would think it'd fit. Console and Command, the two features which take up the most space, are both disabled at the keyboard level.

Reply:
Thanks for the explanation!